src/VT100.jl

```diff
@@ -167,7 +167,7 @@ end
 Base.size(s::ScreenEmulator) = (s.ViewPortSize.height, s.ViewPortSize.width)
 create_cell(em::ScreenEmulator,c::Char) = Cell(em.cur_cell, content = c)
 cur_cell(em::ScreenEmulator) = em.cur_cell
-set_cur_cell!(em::ScreenEmulator,c::Cell) = em.cur_cell = c
+set_cur_cell(em::ScreenEmulator,c::Cell) = em.cur_cell = c
```

Comment:
It really should have a `!`. Can we keep both?

Reply:
It is currently inconsistent with https://github.com/Keno/VT100.jl/blob/07d02efc73b00228e4dadac33ffa7691078f609d/src/VT100.jl#L240

Reply:
Let's switch everything to `!`. If necessary, we can keep the non-`!` version for compatibility.

Reply:
Also, I guess we lost CI or it would have caught this. Could you update the CI with whatever the current way we test packages is (github actions I guess?)

Reply:
Made `!` be used consistently and added CI. It is not showing up though, maybe you have to approve it first time?

Reply:
I think it needs to be on master first for it to know to turn it on. I'll merge and see what happens.